bot/cogs/sticky.py

```diff
@@ -0,0 +1,256 @@
+        message: app_commands.Range[str, 0],
+        channel: Union[discord.TextChannel, discord.ForumChannel, discord.Thread]
+    ) -> None:
+        await interaction.response.defer()
+        if interaction.permissions.manage_channels:
+            async with self.db_pool.acquire() as conn:
+                res = await conn.fetch(
+                    "SELECT channel_id,message_id FROM sticky WHERE channel_id = $1;", channel.id
+                )
+            if not res:
+                embed = discord.Embed(
+                    color=discord.Color.red(),
+                    title="❌ Sticky message not exist",
+                    description=f"Tidak ada sticky message pada channel <#{channel.id}>",
+                    timestamp=datetime.now()
+                )
+            else:
+                data = dict(res[0])
+
+                sticky = await channel.fetch_message(data['message_id'])
+                await sticky.edit(content=message)
+
+                async with self.db_pool.acquire() as conn:
+                    await conn.execute(
+                        "UPDATE sticky SET message = $2 WHERE channel_id = $1;", channel.id, message
+                    )
+
+                embed = discord.Embed(
+                    color=discord.Color.green(),
+                    title="✅ Sticky message update successfully",
+                    description=f"Berhasil memperbaharui sticky message pada channel <#{channel.id}>",
+                    timestamp=datetime.now()
+                )
+        else:
+            embed = discord.Embed(
+                color=discord.Color.red(),
+                title="❌ You Don't Have Permission To Delete Sticky Message",
+                description=f"Permission Manage Channel Dibutuhkan",
+                timestamp=datetime.now()
+            )
+
+        embed.set_footer(
+            text=f"{str(interaction.user)}",
+            icon_url=interaction.user.display_avatar.url
+        )
+
+        await interaction.followup.send(embed=embed)
+        
+
+    @commands.guild_only()
+    @app_commands.command(name="remove", description="Remove sticky message from channel")
+    @app_commands.describe(channel="Target Channel")
+    async def remove_sticky_message(
+        self,
+        interaction: Interaction,
+        channel: Union[discord.TextChannel, discord.ForumChannel, discord.Thread]
+    ) -> None:
+        await interaction.response.defer()
+        if interaction.permissions.manage_channels:
+            async with self.db_pool.acquire() as conn:
+                res = await conn.fetch(
+                    "SELECT channel_id,message_id FROM sticky WHERE channel_id = $1;", channel.id
+                )
+            if not res:
+                embed = discord.Embed(
+                    color=discord.Color.red(),
+                    title="❌ Sticky message not exist",
+                    description=f"Tidak ada sticky message pada channel <#{channel.id}>",
```

Comment:
```suggestion
                    description=f"Tidak ada sticky message pada channel {channel.mention}",
```
This can be simplified by calling `mention` property from `Channel` object.